src/main/java/org/spongepowered/api/network/ChannelRegistrar.java

```diff
@@ -35,6 +36,85 @@
  */
 public interface ChannelRegistrar {
 
+    /**
+     * Creates a new login channel binding for the given channel name. The channel can
+     * be used to send and receive messages.
+     *
+     * @param plugin The plugin registering the channel
+     * @param channel The channel to register
+     * @return A new {@link LoginChannelBinding} instance bound to the channel name
+     * @throws ChannelRegistrationException The channel name is too long
+     * @throws ChannelRegistrationException The channel name is reserved
+     */
+    LoginChannelBinding.IndexedMessageChannel createLoginChannel(Object plugin, String channel) throws ChannelRegistrationException;
```

Comment:
This seems like a lot of duplication with the existing API. Perhaps channel bindings could be per phase (login/play)?